examples/api-basic/main.go

```diff
@@ -17,20 +17,35 @@ limitations under the License.
 package main
 
 import (
+	"flag"
+	stdlog "log"
 	"net/http"
 
+	"github.com/caicloud/nirvana"
+
 	"github.com/caicloud/nirvana/examples/api-basic/api/v1"
 	"github.com/caicloud/nirvana/examples/api-basic/api/v2"
+	"github.com/caicloud/nirvana/log"
 	"github.com/caicloud/nirvana/web"
 )
 
 func main() {
+	flag.Parse()
```

Comment:
why do you need `flag.Parse`?

BTW, if you do need flag, then use cli package :)

Reply:
This is a mistake. Should be removed.